modules/augment/include/opencv2/augment.hpp

```diff
@@ -0,0 +1,18 @@
+/// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+/** @defgroup augment Data augmentation for deep learning
+*/
+#ifndef OPENCV_AUGMENT_AUGMENT_HPP
+#define OPENCV_AUGMENT_AUGMENT_HPP
+#include <opencv2/augment/transform.hpp>
+#include <opencv2/augment/flip.hpp>
+#include <opencv2/augment/rotate.hpp>
+#include <opencv2/augment/gaussianblur.hpp>
+#include <opencv2/augment/averageblur.hpp>
+#include <opencv2/augment/resize.hpp>
+#include <opencv2/augment/augmenter.hpp>
+#include <opencv2/augment/croptofixedsize.hpp>
```

Comment:
I think that we need to move all the transformations into a single file and name it somethind like `all_transformations.hpp`.

Reply:
why? I think it is clearer that each transformation has its own file